test/functional/tests/fault_injection/test_max_io_greater_in_core.py

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+import pytest
+
+from api.cas import casadm, cas_module
+from api.cas.cache_config import CacheMode, CacheLineSize
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+from test_tools.dd import Dd
+from test_utils import os_utils
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.parametrizex("module", cas_module.CasModule)
+@pytest.mark.parametrizex("cache_mode", CacheMode)
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.require_plugin("scsi_debug", delay="0", virtual_gb="4", dev_size_mb="500",
+                            sector_size="512", physblk_exp="4")
+def test_max_io_greater_in_core(module, cache_mode, cache_line_size):
+    """
+        title: Test behavior when core's max IO (max_sectors_kb) is greater than cache's.
+        description: |
+          Test behavior when core's max IO (max_sectors_kb) is greater than cache' for
+          various CAS configurations.
+        pass_criteria:
+          - No kernel bug.
+          - Running workload successfully.
+          - Fulfilling criteria: core.max_io > cache.max_io & core.max_io > 1MiB.
+    """
+    with TestRun.step("Prepare devices."):
+        core_scsi_debug_disk = TestRun.scsi_debug_devices[0]
+
+        cache_disk = TestRun.disks['cache']
+        cache_disk.create_partitions([Size(1, Unit.GibiByte)])
+        cache_dev = cache_disk.partitions[0]
+
+        TestRun.executor.run(f"mount /DVD")
```

Comment:
Is it used somewhere in this test? What is the purpose of this line?
Isn't it better to use here `run_expect_success` method?

Reply:
good point, I forgot to run test without this because I am also not sure what is it for and if that makes any difference. Will check this!